server/tournaments/index.ts

```diff
@@ -1115,7 +1048,7 @@ class Tournament extends Rooms.RoomGame {
 		p2.games += 1;
 		if (!(p1.isDisqualified || p2.isDisqualified)) {
 			// If a player was disqualified, handle the results there
-			const error = this.generator.setMatchResult([p1, p2], result, score);
+			const error = this.generator.setMatchResult([p1, p2], result as 'win' | 'loss', score);
```

Comment:
Should this cast be necessary? Should `setMatchResult` take `'draw'`?

Reply:
If `setMatchResult` takes anything other than `win` and `loss` it returns `InvalidMatchResult`, so I don't think it should.

But if there's a winner (by virtue of there being a disqualified player) `result` definitely isn't `draw`, right?